app/helpers/template_helper.py

```diff
@@ -81,5 +81,6 @@ def render_template(template, **kwargs):
         account_service_url=cookie_session.get('account_service_url'),
         account_service_log_out_url=cookie_session.get('account_service_log_out_url'),
         survey_id=g.schema.json['survey_id'],
+        survey_guide=g.schema.json.get('survey_guide'),
```

Comment:
Personally I think this should be named `external_guidance_url` to make it clear what is being passed into the template.